lib/ui/button.tsx

```diff
@@ -42,6 +42,39 @@ const buttonVariants = cva(
   }
 )
 
+const iconVariants = cva("transition-colors", {
+  variants: {
+    variant: {
+      default: "text-f1-icon-inverse-secondary",
+      outline: "text-f1-icon",
+      neutral: "text-f1-icon",
+      critical:
+        "text-f1-icon-critical-bold group-hover:text-f1-icon-inverse-secondary",
+      ghost: "text-f1-icon",
+      promote: "text-f1-icon",
```

Comment:
It's a shame that we're losing dark theme colors, but I guess we're going to work on it as a separate epic months (if not years) from now.

Reply:
Ah, no! We're not, It's just that the dark colors are currently the same as the light ones. However, the theme system is in place and ready for when we define them at a design level.